### ISSUE TITLE
When can the tensorrtllm container be released?

### ISSUE THREAD

**Issue by insist93** [NVIDIA/TensorRT-LLM#725]
When can the tensorrtllm container be released? I have failed to build docker environment due to network problems. Thank you.

**Comment by Shixiaowei02** [comment]
If you are having difficulty accessing GitHub, please use the following commands to install TensorRT-LLM in a docker container. Thank you!

```
make -C docker devel_build
docker run -it tensorrt_llm bash
pip3 install tensorrt_llm -U --extra-index-url https://pypi.nvidia.com
```

**Comment by miboettc** [comment]
The wheels are only available for Python 3.10. Any plans to provide them also for 3.9 or 3.11?

**Comment by Shixiaowei02** [comment]
> The wheels are only available for Python 3.10. Any plans to provide them also for 3.9 or 3.11?

We recommand using TensorRT-LLM with Python 3.10. If are unable to switch this version of Python, please try installing TensorRT-LLM from source code. Thank you!
